main.py

```diff
@@ -47,6 +47,17 @@ def add_card():
     data_manager.add_new_board('board' + next_board_num)
 
 
+@app.route("/change-board-title", methods=['PUT'])
+@json_response
+def change_board_title():
+    req = request.get_json()
+    print(req)
```

Comment:
I think we don't need this print here. Btw. the _debugger_ is a more flexible and more useful alternative of _printing things to the console_. :)